### PR TITLE
Add quadkey module to redshift

### PR DESCRIPTION
**Pull request by vdelacruzb**
- Add quadkey functions:
  - The functions returning arrays KRING, TOCHILDREN, ST_ASQUADINT_POLYFILL are returning SUPER, that should work like them.
  - The functions returning Geometry ST_BOUNDARY are returning VARCHAR for now. If Redshift doesn't create a `ST_GEOMFROMGEOJSON`, we will convert geojsons to wkb/wkt.
  - As in SQL functions we don't have named parameters I'm leaving a comment just below, maybe we can use that style or similar.
  - The polyfill function supports Geometry Collections as in other warehouses. The tilecover code as been ported from https://github.com/mapbox/tile-cover and https://github.com/mapbox/tilebelt.
  - Mercantile already contained functions for parent, children, neighbors but used the same that we created in other warehouses for convinience.
 
- Modified the Makefile:
  - So we can create structure of folders inside lib, and not insert everything in the same __init__.py
  - Added -rP to pytest in case we want to debug the test and see the prints.
  - If found venv2 installs `python2_module_requirements.txt` which are the module specific dependencies, used for building the library and unit tests.
  - If found venv3 installs `python3_module_requirements.txt` which are the module specific dependencies used only for integration tests.
  - Library installer runs through venv3 and install the modules in `python2_module_requirements`.

[ch176381]

### CODE REVIEW

**Comment by vdelacruzb** [comment]
BTW when we merge this PR, we should also take a look to advanced to update the MAKEFILE, add library_installer, etc.

File: modules/quadkey/redshift/sql/LONGLAT_ASQUADINT.sql [hunk 1]
```diff
@@ -0,0 +1,17 @@
+----------------------------
+-- Copyright (C) 2021 CARTO
+----------------------------
+
+CREATE OR REPLACE FUNCTION @@RS_PREFIX@@quadkey.LONGLAT_ASQUADINT
+(longitude FLOAT8, latitude FLOAT8, resolution INT)
+-- (longitude, latitude, resolution)
```
Comment (Jesus89) [inline modules/quadkey/redshift/sql/LONGLAT_ASQUADINT.sql:7]: Comment not needed

File: modules/quadkey/redshift/sql/ST_ASQUADINT_POLYFILL.sql [hunk 2]
```diff
@@ -0,0 +1,43 @@
+----------------------------
```
Comment (Jesus89) [inline modules/quadkey/redshift/sql/ST_ASQUADINT_POLYFILL.sql:1]: Remove one copyright header

**Review by Jesus89 — COMMENTED** [review]
Outstanding work!

I have left some comments to review. Tomorrow I'll test it in staging

**Comment by Jesus89** [comment]
The tests fail because `storage-upload` requires `venv3`. I'll upload the fix

**Review by Jesus89 — APPROVED** [review]
LGTM!

**Comment by Jesus89** [comment]
Re-testing staging deployment

**Review by Jesus89 — APPROVED** [review]

**Pull request merged**